src/components/molecules/Jukebox/Jukebox.scss

```diff
@@ -0,0 +1,96 @@
+@import "scss/constants.scss";
+
+$emoji-button-diameter: $submit-button-diameter - $spacing--md;
+$jukebox-chat-max-height: 100px;
+
+.Jukebox {
+  &__container {
+    width: 100%;
+    height: 35%;
+    margin: $spacing--lg;
+  }
+
+  &__chat {
+    overflow-y: scroll;
+    display: flex;
+    flex-direction: column;
+    max-height: $jukebox-chat-max-height;
+    background-color: opaque-white(0.12);
+    height: 100%;
+    width: 100%;
+    border-top-left-radius: $border-radius--xl;
+    border-top-right-radius: $border-radius--xl;
+
+    &-messages {
+      padding: $spacing--xs $spacing--md;
+    }
+  }
+
+  &--question {
+    background-color: $dark-green;
+  }
+
+  &__form {
+    padding: $spacing--md 0;
+    display: flex;
+  }
+
+  &__inputs {
+    display: flex;
+    align-items: center;
+    place-content: center;
+    flex-grow: 1;
+    margin-right: $spacing--lg;
+  }
+
+  &__input-container {
+    flex-grow: 1;
+    display: flex;
+
+    > div {
```

Comment:
let's give a class name to this div if possible. 🙃

Reply:
done d04b6de